go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/abdullin/seq v0.0.0-20160510034733-d5467c17e7af // indirect
 	github.com/aws/aws-sdk-go v1.25.41
 	github.com/denverdino/aliyungo v0.0.0-20170926055100-d3308649c661
-	github.com/digitalocean/godo v1.1.1
+	github.com/digitalocean/godo v1.7.5
```

Comment:
This is the lowest version which included this fix, so that consumers of this library are not forced to upgrade their dependencies beyond what is necessary.